packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -206,6 +209,30 @@ const genFullBackCompatConfig = (): CompatConfig[] => {
 	return _configList;
 };
 
+/**
+ * Returns true if compat test version is below the one provided as minimum version.
+ * It helps to filter lower verions configs that the ones intended to be tested on a
+ * particular suite.
+ * @param minVersion -
+ * @param config -
+ * @returns boolean
+ */
+export function isCompatVersionBelowMinVersion(minVersion: string, config: CompatConfig) {
+	let lowerVersion: string | number;
```

Comment:
nit: You can do `let lowerVersion = config.compatVersion;` here. That way you don't need the else block below. Just a small optimization.